lib/livebook/hubs/dockerfile.ex

```diff
@@ -116,20 +116,31 @@ defmodule Livebook.Hubs.Dockerfile do
     RUN /app/bin/warmup_apps.sh
     """
 
+    random_secret_key_base = Livebook.Utils.random_secret_key_base()
+    random_cookie = Livebook.Utils.random_cookie()
+
     startup =
       if config.clustering == :fly_io do
-        ~S"""
-        # Custom startup script to cluster multiple Livebook nodes on Fly.io
-        RUN printf '\
-        #!/bin/bash\n\
-        export ERL_AFLAGS="-proto_dist inet6_tcp"\n\
-        export LIVEBOOK_NODE="${FLY_APP_NAME}-${FLY_IMAGE_REF##*-}@${FLY_PRIVATE_IP}"\n\
-        export LIVEBOOK_CLUSTER="dns:${FLY_APP_NAME}.internal"\n\
-        /app/bin/livebook start\n\
-        ' > /app/bin/start.sh && chmod +x /app/bin/start.sh
-
-        CMD [ "/app/bin/start.sh" ]
         """
+        # --- Clustering ---
+
+        # Set the same Livebook secrets across all nodes
+        ENV LIVEBOOK_SECRET_KEY_BASE "#{random_secret_key_base}"
+        ENV LIVEBOOK_COOKIE "#{random_cookie}"
```

Comment:
We don't need LIVEBOOK_COOKIE, do we? changing it on deployments is completely fine and perhaps even desired, no?